aicsimageio/tests/readers/test_arraylike_reader.py

```diff
@@ -32,42 +32,12 @@ def test_arraylike_reader(arr, expected_shape, expected_dims):
     assert reader.metadata is None
     assert reader.shape == expected_shape
     assert reader.dask_data.shape == expected_shape
-    assert reader.size(expected_dims) == expected_shape
+    assert reader.get_size(expected_dims) == expected_shape
 
     # Will error because those dimensions don't exist in the file
     with pytest.raises(exceptions.InvalidDimensionOrderingError):
-        assert reader.size("ABCDEFG") == expected_shape
+        assert reader.get_size("ABCDEFG") == expected_shape
 
     # Check array
     assert isinstance(reader.data, np.ndarray)
     assert reader.data.shape == expected_shape
-
-
-@pytest.mark.parametrize(
-    "expected_starting_dims, set_dims, expected_ending_dims",
```

Comment:
Do we lose coverage by removing these or were they completely redundant?

Reply:
The only thing this tested for was custom dims setting which has been removed entirely so the coverage diff should be disregarded in situations like this.